test/wdio-shared.conf.js

```diff
@@ -97,10 +97,19 @@ exports.config = {
     wdi5: {
         screenshotPath: path.join('test', 'report', 'screenshots'),
         deviceType: 'native',
-        logLevel: 'error',
+        logLevel: 'verbose',
         capabilities: {
             rotate: true,
             camera: 2
+        },
+        plugins: {
```

Comment:
the plugins section is both in wdio-shared.conf.js + wdio.conf.js - wouldn't it be sufficient in only the shared config?

Reply:
wdio-shared.conf.js is for native tests eg. ios, android... the wdio.conf.js is for browser testing.